### DISCUSSION TITLE
Data Asset Management

### DISCUSSION THREAD

**Discussion by ChenpiDog** [dataplane-app/dataplane#453] [Ideas & feature requests]
1) Data map: It shows the data distribution of various data sources, such as how many databases, tables, rows, and data volumes in a GreenPlum type data connection.
2) Data directory: Allows users to group tables (multi tier) for systematic management of tables. Tables are obtained by reading data connections. They are real, not imaginary.
3) Data dictionary: The data dictionary is provided through pdf (or other methods), including database name, table name, field name, field type, field constraint, table comment, field comment and other information.

**Comment by saul-data** [comment]
It would be good to see some examples on how this benefits enterprise customers. Happy to setup a call to discuss. :)

**Reply by ChenpiDog** [reply]
This is not clear in a few words. We can talk about this topic for a long time. You can learn about data governance first.

**Reply by ChenpiDog** [reply]
Data asset management is a part of data governance. Data governance also includes Master Data Management (MDM), Metadata Management (lineage data kinship is inside), Data Quality, Data Security, etc. Of course, these are all conceptual things. How to implement them into products and how to get the optimal solution need to be considered.

**Reply by ChenpiDog** [reply]
I just put forward some ideas on Dataplane from the aspects of data warehouse and data governance.
